docs/install/registries.md

```diff
@@ -25,6 +25,92 @@ To configure a private registry scoped to a particular organization:
 "@myorg3" = { token = "$npm_token", url = "https://registry.myorg.com/" }
 ```
 
+### Changing the Mirror Source in Bun
```

Comment:
Why creating another section when there's already this?
https://github.com/oven-sh/bun/blob/214b3ccca052203211e18f9d55796549c135edf3/docs/install/registries.md?plain=1#L1-L11

Also I don't see any reason why we should maintain list of available npm mirrors. We're recommending them but have no guarantee that they are/will be OK.

Reply:
or maybe we can update the instructment with real registry like `npmmirror` ?

Reply:
npmjs is not a real registry? Or what do you mean?

Reply:
plz check the update. It's what I mean